### ISSUE TITLE
Add more themes or respect the system theme

### ISSUE THREAD

**Issue by mohegan** [fluoroom/fluoroom-gnome-top-bar#1]
Thanks for your work. It looks geat !

Just a little request : can you add more themes or respect the current gnome-theme, please ?
I use ubuntu and the color of the top bar is too dark.

**Comment by fluoroom** [comment]
Hi! Thanks for trying my work and proposing ideas!

I have plans of adding the option to choose background and text (icons) colors. Reading the colors that the current theme is using can be another option, but it may not be 100% accurate with all themes, so i MAY not include it.
(I assume your idea of "adding themes" is referred to background and foreground colors.)

Also, this is the style i (fluoroom) like: black background and round corners make the illussion of the bar's content being outside the screen. Changing this would convert "fluoroom's Top Bar" to "Color Customizable Persistent Gnome's Default Top Bar".

I would't mind if that happens (i prefer people enjoy it than imposing my black-background style) and yes, the name would stay the same.
For now i'm pretty happy with this and i'm continuing my work on other extensions and personal projects, so please don't expect speed.

TL;DR:
Good ideas, added to the list, just don't expect them for June.
